### DISCUSSION TITLE
Memos Default Interface: 9030? Why not 5230?

### DISCUSSION THREAD

**Discussion by albotroz** [community-scripts/ProxmoxVE#274] [Q&A] [answered]
In the Memos docs it says that the default port is 5230, but for the Memos LXC container it says 9030.

Are there any reasons why?
https://community-scripts.github.io/ProxmoxVE/scripts?id=memos

**Comment by MickLesk** [comment, marked as answer]
Nope, there is no reason for this, except that ports 4000-6500 are blocked and I always make developments above > 8000.

Doesn't change the logic :-D
